<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="loginLabel">
      <property name="styleSheet">
       <string>
        QLabel {
            font-size: 16pt;
            font-weight: bold;
            color: #333;
        }
       </string>
      </property>
      <property name="text">
       <string>Login to Your Account</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="emailLineEdit">
      <property name="styleSheet">
       <string>
        QLineEdit {
            padding: 10px;
            border: 1px solid #aaa;
            border-radius: 5px;
        }
       </string>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="styleSheet">
       <string>
        QLineEdit {
            padding: 10px;
            border: 1px solid #aaa;
            border-radius: 5px;
        }
       </string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loginButton">
      <property name="styleSheet">
       <string>
        QPushButton {
            background-color: #0078d7;
            color: white;
            padding: 10px;
            border-radius: 5px;
        }
        QPushButton:hover {
            background-color: #005fa3;
        }
       </string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="registerButton">
      <property name="styleSheet">
       <string>
        QPushButton {
            background-color: #28a745;
            color: white;
            padding: 10px;
            border-radius: 5px;
        }
        QPushButton:hover {
            background-color: #218838;
        }
       </string>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
